modules/tree/src/main/tree.scala

```diff
@@ -83,7 +314,7 @@ case class Branch(
     comments: Node.Comments = Node.Comments(Nil),
     gamebook: Option[Node.Gamebook] = None,
     glyphs: Glyphs = Glyphs.empty,
-    children: List[Branch] = Nil,
+    children: Branches = Branches.empty, // Vector used in `Study.Node`, switch?
```

Comment:
⚠️ Previously study node children were stored as vector, but I couldn't figure why this was needed. Sure, editing the mainline node is now `O(n)` instead of `O(1)` but given there're usually very few variations it should be insignificant.

Reply:
I think we mostly look up item, by traverse the list item by item, so it's even probably better with List, I guess.
Check:
https://www.lihaoyi.com/post/BenchmarkingScalaCollections.html#vectors-are-ok